hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/api/db.json

```diff
@@ -13,7 +14,8 @@
     "openContainers": 5,
     "volumes": 5,
     "buckets": 156,
-    "keys": 253000
+    "keys": 253000,
+    "deletedKeys": 1000
```

Comment:
maybe use the `pendingDeleteKeys` here as well, so the naming won't confuse anyone reading the code

Reply:
Done Changes as suggested by you from frontend.